.github/actions/run-tests/Dockerfile

```
@@ -3,7 +3,7 @@ ARG GO_VERSION=1.16.12
 ARG XX_VERSION=1.1.0
 
 ARG LIBGIT2_IMG=ghcr.io/fluxcd/golang-with-libgit2
-ARG LIBGIT2_TAG=libgit2-1.1.1-1
+ARG LIBGIT2_TAG=sha-a926b575d640a7f99876f0d1768179f6b586d816
```

```suggestion
ARG LIBGIT2_TAG=libgit2-1.1.1-2
```

(Pushed the tag now that we've confirmed things to be working on this end)